client-network/src/network_ops/default_network_ops_client.rs

```diff
@@ -32,6 +48,86 @@ impl<'a, W> NetworkOpsClient for DefaultNetworkOpsClient<'a, W>
 where
     W: WalletClient,
 {
+    fn create_deposit_bonded_stake_transaction(
+        &self,
+        name: &str,
+        passphrase: &SecUtf8,
+        from_address: &ExtendedAddr,
+        inputs: Vec<TxoPointer>,
+        to_staked_account: StakedStateAddress,
+        attributes: StakedStateOpAttributes,
+    ) -> Result<TxAux> {
+        match from_address {
+            ExtendedAddr::BasicRedeem(ref redeem_address) => {
+                let secp = Secp256k1::new();
+                let transaction: DepositBondTx = DepositBondTx {
+                    inputs,
+                    to_staked_account,
+                    attributes,
+                };
+
+                let public_key = self
+                    .wallet_client
+                    .find_public_key(name, passphrase, redeem_address)?
+                    .ok_or_else(|| Error::from(ErrorKind::AddressNotFound))?;
+
+                let private_key = self
+                    .wallet_client
+                    .private_key(passphrase, &public_key)?
+                    .ok_or_else(|| Error::from(ErrorKind::PrivateKeyNotFound))?;
+                let secret_key: SecretKey = SecretKey::from(&private_key);
+                let witness2 = vec![TxInWitness::BasicRedeem(get_ecdsa_witness(
+                    &secp,
+                    &transaction.id(),
+                    &secret_key,
+                ))]
+                .into();
+
+                Ok(TxAux::DepositStakeTx(transaction, witness2))
+            }
+            ExtendedAddr::OrTree(_) => Err(ErrorKind::InvalidInput.into()),
+        }
+    }
+    fn create_unbond_stake_transaction(
+        &self,
+        name: &str,
+        passphrase: &SecUtf8,
+        from_address: &ExtendedAddr,
+        value: Coin,
+        attributes: StakedStateOpAttributes,
+    ) -> Result<TxAux> {
+        match from_address {
+            ExtendedAddr::BasicRedeem(ref redeem_address) => {
+                let secp = Secp256k1::new();
+                let transaction: UnbondTx = UnbondTx {
+                    value,
+                    nonce: 0,
+                    attributes,
+                };
+
+                let public_key = self
+                    .wallet_client
+                    .find_public_key(name, passphrase, redeem_address)?
+                    .ok_or_else(|| Error::from(ErrorKind::AddressNotFound))?;
+
+                let private_key = self
+                    .wallet_client
+                    .private_key(passphrase, &public_key)?
+                    .ok_or_else(|| Error::from(ErrorKind::PrivateKeyNotFound))?;
+                let secret_key: SecretKey = SecretKey::from(&private_key);
```

Comment:
Signing should be done using `sign()` function on `private_key`.

Something similar to this:
```
let signature = private_key
    .sign(transaction.id())
    .map(StakedStateOpWitness::new)?;
```